src/main/java/com/example/planit/engine/CalendarEngine.java

```diff
@@ -1414,6 +1416,23 @@ private static void addBreakTimeInEndOfEachUserEvent(List<Event> userEvents, int
         }
     }
 
+    /**
+     * add break in the start of each user event in the list
+     *
+     * @param userEvents    list of all the {@link Event} the user have in the calendars
+     * @param userBreakTime the break time in user preferences in minutes
+     */
+    private static void addBreakTimeInStartOfEachUserEvent(List<Event> userEvents, int userBreakTime) {
+
+        for (Event currentUserEvent : userEvents) {
+            Instant oldStartOfUserEvent = Instant.ofEpochMilli(currentUserEvent.getStart().getDateTime().getValue());
+            Instant newStartOfUserEvent = oldStartOfUserEvent.minusSeconds(2L * userBreakTime * MINUTES_IN_SECONEDS);
+            currentUserEvent.setStart(new EventDateTime()
```

Comment:
`Instant newStartOfUserEvent = oldStartOfUserEvent.minusSeconds(2L * userBreakTime * MINUTES_IN_SECONEDS);`

why are you giving a break of (2 * userBreakTime) ?
if userBreakTime is big like 60 minutes it would create a window of 2 hours between each event. you sure that is what you want ? 

i dont mind but i believe somewhere we did something similar with Math.max(60, 2 * userBreakTime)
maybe it should be placed here as well